app/src/components/map/components/MarkerClusterGroup.tsx

```diff
@@ -0,0 +1,67 @@
+import { createPathComponent, LeafletContextInterface } from '@react-leaflet/core';
```

Comment:
It doesn't look like anything from here is used anywhere, so probably can remove this whole file?